inst/pages/extra_material.qmd

```diff
@@ -678,9 +679,9 @@ Here, we use following packages:
 -   [biclust](https://cran.r-project.org/web/packages/biclust/index.html)
 -   [cobiclust](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13582)
 
-_cobiclust_ is especially developed for microbiome data whereas _biclust_ is
-more general method. In this section, we show two different cases and example
-solutions to apply biclustering to them.
+`_cobiclust_` is especially developed for microbiome data whereas `_biclust_` is
```

Comment:
These packages should be

\`cobiclust\` and not \`\_cobiclust\_\`

There were couple of lines were this same occurred. Fix them all

Reply:
Sure, the underscores were there initially so I thought not to remove them. Maybe they were also formatting errors, I shall do the needful.

Reply:
Yep, we have used different styles previously, but now we should use harmonized style everywhere

Reply:
Hello, I am no longer able to make changes to these files, should I make the corrections ina new PR?

Reply:
Yes you can do that, probably easier now as this was merged.